test/specs.js

```diff
@@ -174,4 +174,29 @@ describe('Custom handler function', function () {
     })
       .to.throw('[module-alias] Expecting custom handler function to return path.')
   })
+
+    it('should return a path matching the alias', function () {
```

Comment:
Number of spaces don't match with the rest of the code. I restarted the travis build so it should show linting errors. Also could you please wrap it with `describe('getAliasPath', ` I know I'm maybe not doing it myself in this project but still 😄

Reply:
Seems to be resolved my side. Hopefully works out